src/accounts/add/add-dialog-sender.js

```diff
@@ -0,0 +1,272 @@
+const LoggerFactory = require('../../core/logging/logger-factory');
+const logger = LoggerFactory.getLogger(__filename);
+
+const title = 'Important: You’ll give all users access to Logz.io';
+const question =
+  'If you add this account, all workspace users can see information on the account, even if they can’t sign in to Logz.io. Do you still want to add the account?';
+
+const messageWithButtons = {
+  attachments: [
+    {
+      title: title,
+      text: question,
+      callback_id: 'should_open_setup_dialog',
+      attachment_type: 'default',
+      delete_original: true,
+      actions: [
+        {
+          text: 'Add the account',
+          value: 'yes',
+          type: 'button',
+          name: 'answer'
+        },
+        {
+          text: 'Never mind',
+          value: 'no',
+          type: 'button',
+          name: 'answer'
+        }
+      ]
+    }
+  ]
+};
+
+const shouldConfigureAliasForCurrentAccount =
+  ' is your default workspace. We need to configure your alias.';
+
+function getMessageWithButtonsForAliasConfiguration(prefix, question, suffix) {
+  return {
+    attachments: [
+      {
+        title: 'Alias Configuration',
+        text: prefix + question + suffix,
+        callback_id: 'add-alias-to-default',
+        attachment_type: 'default',
+        delete_original: true,
+        actions: [
+          {
+            text: 'Add alias',
+            value: 'yes',
+            type: 'button',
+            name: 'yes'
+          },
+          {
+            text: 'Not now',
+            value: 'no',
+            type: 'button',
+            name: 'no'
+          }
+        ]
+      }
+    ]
+  };
+}
+
+const messageWithoutButtons = {
+  response_type: 'ephemeral',
+  attachments: [
+    {
+      title: title,
+      text: question
+    }
+  ]
+};
+
+function maskApiToken(logzioApiToken) {
+  if (!logzioApiToken) {
+    return '';
+  }
+
+  if (logzioApiToken.length < 6) {
+    return '*'.repeat(logzioApiToken.length);
+  }
+
+  const charsToMask = logzioApiToken.length - 6;
+  return '*'.repeat(charsToMask) + logzioApiToken.substring(charsToMask);
+}
+
+function createSelectableRegionList(apiConfig) {
+  const configuredRegions = apiConfig['regions'];
+  const selectableRegionList = [];
+  for (const region in configuredRegions) {
+    if (!configuredRegions.hasOwnProperty(region)) continue;
+
+    selectableRegionList.push({
+      label: configuredRegions[region]['name'],
+      value: region
+    });
+  }
+
+  return selectableRegionList;
+}
+
+function buildAndSendConfigurationDialog(
+  bot,
+  selectableRegionList,
+  reply,
+  config,
+  callback_id
+) {
+  const accountRegion = config.getLogzioAccountRegion() || 'us-east-1';
```

Comment:
not sure we want to add default value 'us-east-1'
the region always need to be taken from the same config file 
and if we change it in that config file it will change it in all files
if we moving our production to azure we will need to change it in all files not only in the config

Reply:
done

Reply:
v